### PR TITLE
Added django 1.9 support

### PR DESCRIPTION
**Pull request by samifahed**

### CODE REVIEW

**Comment by kmmbvnr** [comment]
thank you! i'll review and merge soon

File: tests/test_templatetags_base.py [hunk 1]
```diff
@@ -22,23 +23,42 @@ def test_get_model_display_data_for_superuser(self):
...
+                    ('Content', 'entity1')],
+                 '/admin/tests/templatetagprocessentity/1/change/'),
+                ('Template Tag Process Entity',
+                 [('Content', 'entity2')],
+                 '/admin/tests/templatetagprocessentity/2/change/'),
+                ('Template Tag Process',
+                 [('Content', 'child_process')],
+                 None)])
+        else:
+            self.assertEqual(data, [
```
Comment (kmmbvnr) [inline tests/test_templatetags_base.py]: that's a problem!

Comment (samifahed) [inline tests/test_templatetags_base.py]: Django changed the admin url for the `change_view`, why should it be a problem?

File: viewflow/templatetags/viewflow.py [hunk 2]
```diff
@@ -4,9 +4,9 @@
 from django.contrib.admin.templatetags.admin_modify import submit_row
 from django.core.exceptions import ImproperlyConfigured
 from django.core.urlresolvers import reverse
-from django.template.base import TemplateSyntaxError, TagHelperNode, parse_bits
+from django.template.library import TemplateSyntaxError, TagHelperNode, parse_bits
```
Comment (kmmbvnr) [inline viewflow/templatetags/viewflow.py]: this breaks dj1.6

Comment (samifahed) [inline viewflow/templatetags/viewflow.py]: ok will fix it

**Comment by samifahed** [comment]
@kmmbvnr fixed it, all the tests pass now

**Comment by kmmbvnr** [comment]
Thank you, nice PR, except it breaks dj 1.6

django 1.9 support added, just few warnings still present

**Pull request closed without merging**